src/core/public/doc_links/doc_links_service.ts

```diff
@@ -553,6 +559,11 @@ export interface DocLinksStart {
   readonly DOC_LINK_VERSION: string;
   readonly OPENSEARCH_WEBSITE_URL: string;
   readonly links: {
+    // readonly wazuh: {
+    //   readonly website: string;
+    //   readonly documentation: string;
+
+    // };
```

Comment:
**nitpick**: remove it if it is unneeded.

Reply:
Work in progress. Not ready for review yet.